torch/_inductor/codegen/triton.py

```diff
@@ -721,6 +721,13 @@ def maximum(a, b):
     def where(a, b, c):
         return f"tl.where({a}, {b}, {c})"
 
+    @staticmethod
+    def inline_asm_elementwise(*inputs, asm, dtype=torch.float32):
```

Comment:
What about all the other arguments:

```
Parameters
:
asm – assembly to run. Must match target’s assembly format.

constraints – asm constraints in [LLVM format](https://llvm.org/docs/LangRef.html#inline-asm-constraint-string)

args – the input tensors, whose values are passed to the asm block

dtype – the element type(s) of the returned tensor(s)

is_pure – if true, the compiler assumes the asm block has no side-effects

pack – the number of elements to be processed by one instance of inline assembly

_builder – the builder
```

Would it make sense to expose any of them in api ?

Reply:
To be clear, this is currently not a "public" API - you can only access it by registering a custom lowering and calling `ops`. In principle, some of these might make sense, but I don't really understand what most of them would be used for, so I only expose `asm` and `dtype` for now.

Reply:
Maybe I should just expose them in-case somebody wants to use them 🤔